index.js

```diff
@@ -3,18 +3,21 @@ var path = require('path');
 var fs = require('graceful-fs');
 var mkdirp = require('mkdirp');
 
-module.exports = function (src, dest, opts, cb) {
+module.exports = function (src, dest, options, cb) {
 	if (!src || !dest) {
 		throw new Error('`src` and `dest` required');
 	}
 
-	if (typeof opts !== 'object') {
-		cb = opts;
-		opts = {};
+	if (typeof options !== 'object') {
+		cb = options;
+		options = {};
 	}
 
 	cb = cb || function () {};
 
+	var opts = {};
```

Comment:
Instead of changing the initial value to `options`, just use [`objectAssign`](https://github.com/sindresorhus/object-assign) to copy over the options to a new object.

```js
opts = objectAssign({}, opts);
```